### PR TITLE
Add command for creating/updating labels

### LINKED ISSUE #12: Need a helper to populate all mentioned github labels
So similar one to `auto`, but now operating on the configs found in the repo and populating also CHANGELOG-missing

### PR DESCRIPTION
**Pull request by jwodder**
Closes #12.

### CODE REVIEW

**Review by yarikoptic — CHANGES_REQUESTED** [review]
make congruent with #17 in just asking the path to the repo?

**Comment by yarikoptic** [comment]
@jwodder, I've added `release` label to see in action, but feel welcome to remove if you would like to test more before kicking it out.

File: README.md [hunk 1]
```diff
@@ -221,3 +258,20 @@ jobs:
...
+
+# Command: `labels`
+
+This repository also provides a command for creating or updating the GitHub
+labels used by the actions and their workflows in a repository.  The
+recommended way to invoke this command is via
+[nox](https://pypi.org/project/nox/); after installing nox, the "labels"
+command can be invoked by running the following in a clone of this repository:
+
+    GITHUB_TOKEN=... nox -e labels -- repo-owner/repo-name path/to/config/file
```
[the remark this answers is not in the document]
Comment (jwodder) [reply README.md:271]: The config file name is not hardcoded, it's just the default; the action can be run with any config file path given as an input.

**Comment by jwodder** [comment]
@yarikoptic Currently, the first release will be v0.1.0, and a v0 tag will be created as a synonym.  Is that OK, or would you prefer to start at v1 like most actions?

**Comment by yarikoptic** [comment]
`v0` sounds great!

**Pull request merged**